onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -517,6 +517,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "The value to be filled in the attention mask. Default value is -10000.0f",
               AttributeProto::FLOAT,
               OPTIONAL_VALUE)
+        .Attr("do_rotary",
```

Comment:
We can rename it as "rotary". I think the rotary is adj, and `do` is not a good fit for it.